torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -314,3 +355,46 @@ def resized_crop_image_pil(
 ) -> PIL.Image.Image:
     img = crop_image_pil(img, top, left, height, width)
     return resize_image_pil(img, size, interpolation=interpolation)
+
+
+def zoom_out_image_tensor(
+    image: torch.Tensor,
+    left: int,
+    top: int,
+    right: int,
+    bottom: int,
+    fill: List[float] = (0.0,),  # type: ignore[assignment]
+) -> torch.Tensor:
+    num_channels, height, width = get_dimensions_image_tensor(image)
+
+    # PyTorch's pad supports only integers on fill. So we need to overwrite the colour
+    output = pad_image_tensor(image, [left, top, right, bottom], fill=0, padding_mode="constant")
+
+    if not isinstance(fill, (list, tuple)):
+        fill = [fill] * num_channels
+    fill = torch.tensor(fill).to().view(-1, 1, 1)
+
+    output[..., :top, :] = fill
+    output[..., :, :left] = fill
+    output[..., (top + height) :, :] = fill
+    output[..., :, (left + width) :] = fill
```

Comment:
Shall we move this fix into main `pad_image_tensor` (aka the `_FT.pad`)? This makes sense as multiple other transforms will need padding (for example the `RandomCrop`). If the underlying pad method respects fill, then this workaround won't be needed in all transforms.

To avoid affecting performance, we should apply this workaround only when `any(x != 0 for x in fill)`

Reply:
Prototype for this is in #5568. I'll hold further development on this PR until that one is done.

Reply:
I think we should resume and put the patch on the transforms. Adding it on the functional JIT-scriptable kernel will lead to very brittle code :(